Dockerfile.runtime

```diff
@@ -4,17 +4,21 @@ FROM centos:centos7
 ARG OPENRESTY_RPM_VERSION="1.13.6.1"
 
 ENV HOME=/opt/app-root/src \
-    PATH=/opt/app-root/src/bin:/opt/app-root/bin:$PATH
+    PATH=/opt/app-root/src/bin:/opt/app-root/bin:$PATH \
+    LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/opt/app-root/src/lib/"
```

Comment:
if `/opt/app-root` should be something like `/` of the app then the libraries should be in `/opt/app-root/lib` as `/opt/app-root/src` should have the app source.